test/index.js

```diff
@@ -19,9 +19,36 @@ import remarkStringify from 'remark-stringify'
 import {assert} from 'mdast-util-assert'
 import {removePosition} from 'unist-util-remove-position'
 import {one, all, defaultHandlers, toMdast} from '../index.js'
+import {wrapNeeded} from '../lib/util/wrap.js'
 
 const fixtures = path.join('test', 'fixtures')
 
+test('custom nodes', (t) => {
+  t.deepEqual(
+    wrapNeeded([
+      {type: 'text', value: 'some '},
+      {
+        // @ts-ignore - custom node type
```

Comment:
```suggestion
        // @ts-expect-error - custom node type
```

expect-error is better!